examples/update_flow_control/update_flow_control.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""example application to control the Update flow of the Mender client
+"""
+
+import os
+import sys
+import threading
+from datetime import datetime, timedelta
+from queue import Empty, SimpleQueue
+from typing import Any, Tuple
+
+from gi.repository.GLib import Error as DBusError  # type: ignore
+from pydbus import SystemBus  # type: ignore
+
+UPDATE_CONTROL_MAP_PAUSE_ALL = """{
+	"priority": -1,
```

Comment:
Should we remove the `priority`?

Reply:
You can include it. This is the best place to show it off I think. But then you should explain what it does. Take a look at [this comment](https://github.com/mendersoftware/mender-docs/pull/1473#discussion_r652472821) (never mind that it was removed in that PR, it's just because it's high level).

This also relates to your question below about UUID. What I would suggest is to add a section, "Going to production", where you highlight some things:

* "Note on Python multi-threading" could be moved here.
* The note about error handling too
* Selecting a UUID. Basically a new random one which stays constant for that application.
* How `priority` interacts with a server based Update Control Map, or one from a different process.
* What happens on map expiry, how `on_map_expire` works, and why refreshing is necessary.

Reply:
Could we not have this as the base-map with priority `0`, or similar, and then the other maps could have `Priority=1`, and then we can remove all but the state they are overriding?
So:
```    
    UPDATE_CONTROL_MAP_CONTINUE_INSTALL = """{
    	"priority": 1,
    "states": {   
    		"ArtifactInstall_Enter": {
    			"action": "continue"
    		},
    	},
    	"id": "01234567-89ab-cdef-0123-456789abcdef"
}
```

Reply:
@kacf Okey about all that points, I added them to a new section.

@oleorhagen I think makes the example code a bit more advance. I have however added a not in the new section on how to use priorities and that the user might want to set different priorities on different maps.